src/client.cpp

```diff
@@ -1307,21 +1311,27 @@ void Client::sendPlayerPos()
 
 	u8 camera_fov    = map.getCameraFov();
 	u8 wanted_range  = map.getControl().wanted_range;
+	float pitch = myplayer->getPitch();
+	float yaw = myplayer->getYaw();
+	if (map.getCameraFrontView()) {
+		yaw += 180 * 100;
```

Comment:
Should be `180.0` in this place, without multiplier. Discussion reference: http://irc.minetest.net/minetest-dev/2017-03-23#i_4837454